File: R/problem_001.R
problem_001 = function(n){
  sum = 0
  for(i in 0:(n-1)){
    if(i %% 3 == 0 | i %% 5 == 0){
      sum = sum + i
    }
  }
  return(sum)
}

problem_001(1000)

File: Java/Problems/problem_001.java

public class problem_001 {

	public static void main(String[] args) {
		System.out.println(solution(1000));
	}
	
	public static int solution(int m) {
		int n = 0;
		for(int i = 0; i < m; i++) {
			if(i % 3 == 0 | i % 5 == 0) {
				n += i;
			}
		}
		return n;
	}
}
